doc/api/stream.md

```diff
@@ -1459,9 +1458,10 @@ write succeeded.
 
 All calls to `writable.write()` that occur between the time `writable._write()`
 is called and the `callback` is called will cause the written data to be
-buffered. Once the `callback` is invoked, the stream will emit a [`'drain'`][]
-event. If a stream implementation is capable of processing multiple chunks of
-data at once, the `writable._writev()` method should be implemented.
+buffered. When the `callback` is invoked, the stream maybe will emit a
```

Comment:
Emitting a 'drain' event need to meet more conditions. See [the source code of writable stream](https://github.com/nodejs/node/blob/master/lib/_stream_writable.js#L482).

Reply:
`maybe will` -> `might`

Reply:
fixed.